great_expectations/rule_based_profiler/config/base.py

```diff
@@ -506,16 +506,16 @@ def __init__(
         BaseYamlConfig.__init__(self, commented_map=commented_map)
 
     @classmethod
-    def from_commented_map(cls, commented_map: CommentedMap):  # type: ignore[no-untyped-def]
+    def from_commented_map(cls, commented_map: CommentedMap):  # type: ignore[override] # super type accepts Dict
         """Override parent implementation to pop unnecessary attrs from config.
 
         Please see parent BaseYamlConfig for more details.
         """
         try:
-            schema_instance: Any = cls._get_schema_instance()
+            schema_instance = cls._get_schema_instance()
```

Comment:
@Kilo59 Anything better we can do here?  Given that there are only so many Marshmallow Schema types involved in RBP, is that something we can use to not leave this variable without a type hint?  If not, no huge deal.  Thanks.

Reply:
Ah, yes if these have to be a marshmallow schema I should be able to use that here.

I will make the change.

Reply:
🙇

Reply:
Done